Initiating PR for new Azure AD Recommendations API (#19841)
microsoftgraph/microsoft-graph-docs: api-reference/beta/api/recommendation-postpone.md

# recommendation: postpone
Namespace: microsoft.graph

[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]

Postpone action on a [recommendation](../resources/recommendation.md) object to a specified future date and time by marking its **status** as `postponed`. On the date and time provided, Azure AD will automatically update the **status** of the **recommendation** object to `active` again.

## Permissions
One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](/graph/permissions-reference).

|Permission type|Permissions (from least to most privileged)|
|:---|:---|
|Delegated (work or school account)|DirectoryRecommendations.ReadWrite.All|
|Delegated (personal Microsoft account)|Not supported.|
|Application|DirectoryRecommendations.ReadWrite.All|

## HTTP request

<!-- {
  "blockType": "ignored"
}
-->
``` http
POST /directory/recommendations/{recommendationId}/postpone
```

## Request headers
|Name|Description|
|:---|:---|
|Authorization|Bearer {token}. Required.|
|Content-Type|application/json. Required.|

## Request body
In the request body, supply a JSON representation of the parameters.

> The following table shows the parameters that can be used with this action.

Reviewer: ```suggestion
The following table shows the parameters that are required with this action.
```

Reviewer: @hafowler - bump

|Parameter|Type|Description|
|:---|:---|:---|

> |postponeUntilDateTime|DateTimeOffset|**TODO: Add Description**|

Reviewer: Update description

Reviewer: @hafowler - bump
